pkgs/applications/misc/nb/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, lib, fetchFromGitHub, installShellFiles
+, bash, git
+}:
+stdenv.mkDerivation rec {
+  version = "5.7.8";
+  pname = "nb";
+
+  src = fetchFromGitHub {
+    owner = "xwmx";
+    repo = "nb";
+    rev = version;
+    sha256 = "1sw09sgskzsn3dy4ipvl52hwh5k3cwqfhjnm7rmxbzcpfk3jm3sp";
+  };
+
+  buildInputs = [ bash git ];
+  nativeBuildInputs = [ installShellFiles ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    mv nb $out/bin/
+    mv bin/bookmark $out/bin/
```

Comment:
```suggestion
    install -Dm755 nb bin/bookmark -t $out/bin
```

Reply:
Thanks, I'll incorporate this with other changes I'm making. I still need to decide whether or not to enable installing all the various optional dependencies from the package.